packages/jest-worker/src/workers/WorkerAbstract.ts

```diff
@@ -125,7 +125,7 @@ export default abstract class WorkerAbstract
    * killed off.
    */
   protected _shutdown(): void {
-    this.state === WorkerStates.SHUT_DOWN;
+    this.state = WorkerStates.SHUT_DOWN;
```

Comment:
This one looks like a mistake.

Reply:
Turns out change to `=` will cause test failue, I'll leave it alone 

https://github.com/jestjs/jest/actions/runs/13658744730/job/38184583400#step:7:61